UPGRADE.md

```diff
@@ -118,6 +118,7 @@
     * `PromotionInterface::setStartsAt`
     * `PromotionInterface::setEndsAt`
     * `PromotionRuleInterface::setPromotion`
+* `createPaginatorForPromotion(string $promotionCode)` added to `PromotionCouponRepositoryInterface`
```

Comment:
And `findByPromotionCode` was removed.

Reply:
Does it? There was no such a method in the beta3 version of Sylius, so it should not be mentioned that it was removed 💎